integration-tests/tests/run-notebook.js

```diff
@@ -0,0 +1,52 @@
+const { withRegisteredUser, withBilling, withWorkspace } = require('../utils/integration-helpers')
+const { testUrl } = require('../utils/integration-config')
+const { click, clickable, signIntoTerra, findElement, waitForNoSpinners, select, delay, fillIn, input, findIframe, findText, dismissNotifications } = require(
+  '../utils/integration-utils')
+
+
+const notebookName = 'TestNotebook'
+
+const testRunNotebookFn = withRegisteredUser(async ({ page, context, email, token }) => {
+  await withBilling(withWorkspace(async ({ workspaceName }) => {
+    await page.goto(testUrl)
+    await click(page, clickable({ textContains: 'View Workspaces' }))
+    await signIntoTerra(page, token)
+    await dismissNotifications(page)
+    await click(page, clickable({ textContains: 'Accept' }))
+    await findElement(page, clickable({ textContains: workspaceName }))
+    await waitForNoSpinners(page)
+    await click(page, clickable({ textContains: workspaceName }))
+    await click(page, clickable({ text: 'notebooks' }))
+    await click(page, clickable({ textContains: 'Create a' }))
+    await fillIn(page, input({ placeholder: 'Enter a name' }), notebookName)
+    await select(page, 'Language', 'Python 2')
+    await click(page, clickable({ text: 'Create Notebook' }))
+    await click(page, clickable({ textContains: notebookName }))
+    await click(page, clickable({ text: 'Edit' }))
+    await select(page, 'Select Environment', 'Hail')
+    await click(page, clickable({ text: 'Create' }))
+    await findElement(page, clickable({ textContains: 'Creating' }))
+    await findElement(page, clickable({ textContains: 'Running' }), { timeout: 3000000 })
+    await delay(5000) //wait for copy over or change of status
+
+    const frame = await findIframe(page)
+    await fillIn(frame, '//textarea', 'print(123456789099876543210990+9876543219)')
```

Comment:
I chose this obnoxiously long number so that is longer than both the test user id and the test workspace id so there won't be a false positive by either of those numbers